rfc/0029-community-amendment-1.md

```diff
@@ -0,0 +1,304 @@
+  community spaces
+* To participate in the processes that shape the Community Team, including voting on new team members and structural
+  changes to community spaces (unless they're a Trainee Community Moderator - see below)
+* To engage with the community authentically, being present and visible in the community when able and appropriate
+* To be supportive of other Community Team members, providing insight, advice and support when able and appropriate
+* To help with the planning and running of community events and projects, when able and appropriate
+
+All team roles are additive - they add additional responsibilities on top of this basic set.
+
+## Team Roles
+
+Community Team members may have specific roles, which define additional responsibilities (and additional tools to meet
+those responsibilities) on top of their basic responsibilities, as defined above. While the roles are intended to be 
+fairly balanced regarding their responsibilities, there is still likely to be some overlap - for example, Community
+Managers also have the *ability* to moderate, if they're required to.
+
+The role structure is intended to be flat, rather than an explicit hierarchy. Roles represent tasks and
+responsibilities, rather than seniority. Roles may be assigned based on a number of criteria, including:
+
+* Previous experience
+* Staffing shortfalls or situational needs
+* Suitability for the role
+* Types of work the team member is offering to handle
+
+### Community Managers
+
+Community managers are responsible for day-to-day administrative tasks in Quilt community spaces, and take an active
+role in the Community Team candidacy process. They're in charge of mobilizing and supporting the rest of the community
+team, maintaining community spaces, and keeping everything running smoothly. Responsibilities include:
+
+* Actioning changes to the structure of community spaces, or to community policies
+* Aiding in conflict resolution and decision-making, especially when it comes to difficult moderation decisions
+* Engaging with the community when they have questions, reports or issues, and working with them to address their
+  concerns
+* Facilitating the Community Team candidacy process by helping other team members to set up polls, and conducting
+  interviews
+* Maintaining informational channels, and writing and posting community-oriented announcements
+* Managing any bots or other community-relevant automation tools
+* Organising, training and supporting the moderation team, and acting as supplemental moderators as required
+* Supporting the rest of the Community Team in their work, providing them with the tools and changes to community 
+  spaces that they need
+
+Community managers must be appointed democratically, following the process outlined later in this document. If
+required, the keyholders will handle handing out the permissions required for community managers to work in Quilt
+community spaces.
+
```

Comment:
Perhaps adding a section for what tools/abilities Community Manager, Moderator, Trainee Moderator, and Admin are given is something that should be done? I'll give an example suggestion under Moderators, since that's the only one I'm completely sure I know what tools I have access to (things like banning, modmail, etc)

Reply:
I think this is something that's likely to change over time, as we open up new spaces and existing spaces change and get updated - maybe we should list it _somewhere_, but I want to avoid having to create a new RFC amendment every time

Reply:
Fair 'nuff